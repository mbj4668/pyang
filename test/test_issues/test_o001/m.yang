module m {
  namespace "urn:test:m";
  prefix m;
  include s;
}
