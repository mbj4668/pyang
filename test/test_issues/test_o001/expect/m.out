m.yang:1: warning: Check module issued for module m
m.yang:1: warning: Check both issued for module m
s.yang:1: warning: Check submodule issued for submodule s
s.yang:1: warning: Check both issued for submodule s
